TYPO3.Neos/Resources/Private/Styles/Media/_Media.scss

```diff
@@ -597,12 +597,11 @@
 				}
 
 				.neos-button-small {
-					height: 32px;
-					width: 32px;
-					padding: 0 10px;
+					height: $wideMargin;
+					width: $wideMargin;
```

Comment:
I generally approve of using variables in SCSS but I don't understand why we make the size of this button depend on a global constant named `$wideMargin` . Maybe we should create our own local variable for that?